### PR TITLE
Add adopter for kueue

### PR DESCRIPTION
**Pull request by GhangZh**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: site/content/en/docs/adopters/index.md [hunk 1]
```diff
@@ -16,3 +16,4 @@ If you are using Kueue, feel free to open a pull request to add your organizatio
 | [CyberAgent, Inc.](https://www.cyberagent.co.jp/en/) | End User | On-premise ML Platform | batch/job </br> kubeflow.org/mpijob | [@tenzen-y](https://github.com/tenzen-y) |
 | [DaoCloud, Inc.](https://www.daocloud.io/en/) | End User | Part of the AI Platform for managing all kinds of Jobs. | batch/job </br> RayJob </br> ... | [@kerthcet](https://github.com/kerthcet) |
 | [WattIQ, Inc.](https://wattiq.io) | End User | SaaS/IoT product | batch/job </br> RayJob </br> | [@madsenwattiq](https://github.com/madsenwattiq) |
+| [Horizon, Inc.](https://horizon.cc/) | End User | AI training platform | batch/job </br> ... | [@kerthcet](https://github.com/kerthcet) |
```
Comment (kerthcet) [inline site/content/en/docs/adopters/index.md:19]: You're the contact actually :)

Comment (GhangZh) [reply site/content/en/docs/adopters/index.md]: OK, I've changed it.

**Review by tenzen-y — COMMENTED** [review]
Thank you!
/lgtm
/approve

**Pull request merged**